src/Android/Resources/values/strings.xml

```diff
@@ -16,4 +16,13 @@
   <string name="PasswordGenerator">
     Password Generator
   </string>
+  <string name="AccessibilityOverlayLine1">
```

Comment:
We should move these into AppResources so that they can be translated. We only use this strings file for a select few things that we cannot use AppResources for.